backend/tests/contributors.spec.ts

```diff
@@ -200,11 +237,33 @@ describe('Contributors unit test', () => {
         });
     });
 
+    describe('initSubItems', () => {
+        it("item has a subitem that does not exist", () => {
+            const item1: IItem = {
+                id: "id1",
+                description: "description1",
+                title: "title1",
+                itemIds: ["itemId1"],
+                labels: []
+            };
+            const res = Contributors.getInstance()["initSubItems"](item1);
+            expect(res).to.be.undefined;
```

Comment:
what do we test here? initSubItems returns void.

Reply:
fixed